x/outline-cli/README.md

```diff
@@ -0,0 +1,32 @@
+# Outline VPN CLI (PoC)
+
+The CLI interface of Outline VPN client for Linux.
+
+## Usage
+
+#### Standard
+
+```
+./OutlineCLI "<svr-ip>" "<svr-port>" "<svr-pass>"
```

Comment:
I think a nicer api would be
```
outline-cli -key=ss://...
```

So it's easier to use given an access key.

You can copy https://github.com/Jigsaw-Code/outline-internal-sdk/blob/main/x/outline-connectivity/main.go